lib/reporting/proofing_rate_report.rb

```diff
@@ -84,55 +86,42 @@ def to_csv
       end
     end
 
+    def sub_reports
+      @sub_reports ||= by_month ? by_month_ranges : trailing_days_ranges
+    end
+
     def reports
       @reports ||= begin
-        sub_reports = [0, *DATE_INTERVALS].each_cons(2).map do |slice_end, slice_start|
-          time_range = if slice_end.zero?
-                         Range.new(
-                           (end_date - slice_start.days).beginning_of_day,
-                           (end_date - slice_end.days).end_of_day,
-                         )
-                       else
-                         Range.new(
-                           (end_date - slice_start.days).beginning_of_day,
-                           (end_date - slice_end.days).end_of_day - 1.day,
-                         )
-                       end
-          Reporting::IdentityVerificationReport.new(
-            issuers: nil, # all issuers
-            time_range: time_range,
-            cloudwatch_client: cloudwatch_client,
-          )
-        end
+        if parallel?
+          threads = sub_reports.map do |report|
+            Thread.new do
+              report.tap(&:data)
+            end.tap do |thread|
+              thread.report_on_exception = false
+            end
+          end
 
-        reports = if parallel?
-                    threads = sub_reports.map do |report|
-                      Thread.new do
-                        report.tap(&:data)
-                      end.tap do |thread|
-                        thread.report_on_exception = false
-                      end
-                    end
-
-                    Reporting::UnknownProgressBar.wrap(show_bar: progress?) do
-                      threads.map(&:value)
-                    end
-                  else
-                    Reporting::UnknownProgressBar.wrap(show_bar: progress?) do
-                      sub_reports.each(&:data)
-                    end
-                  end
```

Comment:
Because we're just returning this, instead of assigning it to `reports`, I was able to unindent everything a bajillion layers.